### PR TITLE
Import Noto script data

### PR DESCRIPTION
**Pull request by simoncozens**
Adam Twardoch pulled together a load of data about each script, which is used to power the script sections in the articles of Noto fonts. It would be helpful if we had this data in the lang repo, where it could be easily updated and edited, as part of making it easier to generate new Noto articles.

### CODE REVIEW

File: Lib/gflanguages/data/scripts/Nand.textproto [hunk 1]
```diff
@@ -1,3 +1,5 @@
 id: "Nand"
 name: "Nandinagari"
-
+historical: true
+family: "Indic"
+summary: "Nandinagari (<span class=\'autonym\'>𑧁𑧞𑧤𑦿𑧁𑧑𑦰𑧈𑧓</span>) is a historical Indic abugida, written left-to-right, with unconnected headstrokes. Was used in the 8th–19th centuries in South India for Sanskrit texts about philosophy, science and the arts. Closely related to Devanagari."
```
Comment (dscorbett) [inline Lib/gflanguages/data/scripts/Nand.textproto:5]: The autonym is “𑧁𑧞𑦿𑧒𑧁𑧑𑦰𑧈𑧓”.

File: Lib/gflanguages/data/scripts/Chrs.textproto [hunk 2]
```diff
@@ -1,3 +1,5 @@
 id: "Chrs"
 name: "Chorasmian"
-
+historical: true
+family: "Middle Eastern"
+summary: "Chorasmian is a historical Middle Eastern abjad, written right-to-left. Was used in the 2nd century BCE–-9th century CE in the Khwarazm region of Central Asia for the now-extinct Chorasmian language, until the language switched to the Arabic script. Derived from Imperial Aramaic."
```
Comment (dscorbett) [inline Lib/gflanguages/data/scripts/Chrs.textproto:5]: There is an extraneous hyphen after the dash in “BCE–-9th”.

File: Lib/gflanguages/data/scripts/Soyo.textproto [hunk 3]
```diff
@@ -1,3 +1,4 @@
 id: "Soyo"
 name: "Soyombo"
-
+family: "Indic"
+summary: "Soyombo (<span class=\'autonym\'>𑪞𑪞‎</span>) is a historical Indic abugida, written left-to-right. Was used in 1686–18th century as a ceremonial and decorative script for the Mongolian language. Also sporadically used for Tibetan and Sanskrit. Created by Bogdo Zanabazar. Needs software support for complex text layout (shaping)."
```
Comment (dscorbett) [inline Lib/gflanguages/data/scripts/Soyo.textproto:4]: This is two instances of the Soyombo symbol, not the Soyombo autonym.

**Comment by simoncozens** [comment]
And this is why I wanted to have this information in a more visible repository. :-)

File: Lib/gflanguages/data/scripts/Dupl.textproto [hunk 4]
```diff
@@ -1,3 +1,4 @@
 id: "Dupl"
 name: "Duployan shorthand"
-
+family: "American"
+summary: "Duployan shorthand (Sloan-Duployan shorthand, Duployan stenography) is an European alphabet, written left-to-right. Geometric stenography script created in 1860 by Father Émile Duployé for writing French, later expanded and adapted for writing English, Chinook Jargon and many others. Heavily cursive (connected), allows words to be written in a single stroke. Praised for simplicity and speed of writing. Needs software support for complex text layout (shaping)."
```
Comment (dscorbett) [inline Lib/gflanguages/data/scripts/Dupl.textproto:4]: “an European” should be “a European”.

Comment (dscorbett) [reply Lib/gflanguages/data/scripts/Dupl.textproto:4]: The `family` should be `"European"`.

Comment (simoncozens) [reply Lib/gflanguages/data/scripts/Dupl.textproto:4]: I'll merge this and fix, just because I'm handling a bunch of changes at ones at the moment.

**Pull request merged**